plugins/woocommerce/src/Admin/Features/Blueprint/RestApi.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+declare( strict_types = 1);
+
+namespace Automattic\WooCommerce\Admin\Features\Blueprint;
+
+use Automattic\WooCommerce\Blueprint\ExportSchema;
+use Automattic\WooCommerce\Blueprint\ImportSchema;
+use Automattic\WooCommerce\Blueprint\JsonResultFormatter;
+use Automattic\WooCommerce\Blueprint\ZipExportedSchema;
+
+/**
+ * Class RestApi
+ *
+ * This class handles the REST API endpoints for importing and exporting WooCommerce Blueprints.
+ *
+ * @package Automattic\WooCommerce\Admin\Features\Blueprint
+ */
+class RestApi {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'blueprint';
+
+	/**
+	 * Register routes.
+	 *
+	 * @since 9.x.x
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/import',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'import' ),
+					'permission_callback' => array( $this, 'check_permission' ),
+				),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/export',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'export' ),
```

Comment:
It should be registered since you have the settings page loaded (feature flag is enabled) 🤔 

I'm unable to reproduce it on my end.

Could you show me the full URL?

Reply:
Thanks, it might've been my local env. I tried again in a fresh JN, it seems to register fine.